docs/resources.rst

```diff
@@ -81,6 +84,9 @@ Editor Integrations
 
     * `Visual Studio Code extension <https://juan.blanco.ws/solidity-contracts-in-visual-studio-code/>`_
         Solidity plugin for Microsoft Visual Studio Code that includes syntax highlighting and the Solidity compiler.
+        
+    * `Solidity Visual Auditor extension <https://marketplace.visualstudio.com/items?itemName=tintinweb.solidity-visual-auditor>`_
+        Ethereum Solidity Language customized for developers and auditors using Visual Studio Code
```

Comment:
This sounds really confusing to me. It says it's a language. I know this line was copied from the original description but it I don't think it explains anything. Is it an extension that adds language support to VSCode? Or a language server?